doc/admin.md

```diff
@@ -14,17 +14,26 @@ If you planned to install Murano from the OpenStack you can follow the oficial
 [OpenStack Murano Installation Guide](http://murano.readthedocs.io/en/stable-liberty/install/index.html)
 
 
-## Installing FIWARE Murano specific requirements
-
-In addition to the oficial Murano documentation, to be FIWARE Murano aware, we need to install some extensions to Murano core library.
-It involves mainly the addition of:
-
--   the FiwareMuranoInstance instance: this is a Murano instance which includes the region and the GE NID (in case required) in the deployed VMs. This is
- required for FIWARE monitoring.
--   the script key: There is a support script which includes a support key to be used by region administrators to give support to users.
--   the murano agent configuration file: It includes different murano-agent configuration files for services to be included in the different
-  linux distributions.
-
+### Installing FIWARE Murano requirements by an script
+fiware-murano repository contains a script which installs all the FIWARE specific requirements. This script is in charge of:
+- creating the virtualenv
+- installing python dependences,
+- generating the core library zip file
+- uploading into murano instance
+
+To execute it, it is required to export admin credentials to access to the Cloud, where Murano is part of.
+
+    $ export OS_USERNAME='admin'
+    $ export OS_PASSWORD={the password for admin user}
+    $ export OS_TENANT_NAME="admin"
+    $ export OS_REGION_NAME={the region}
+    $ export OS_AUTH_URL={the auth url for keystone}
+
+Then, just go to folder scripts and execute the script:
+    $ cd scripts
+    $ ./upload_fiware_things.sh
```

Comment:
This file have to be executed always inside scripts directory. If you keep the script as is, I suggest to add a Note explaining it.